README.md

```diff
@@ -2,35 +2,41 @@
 
 ## General information
 
-VSCode stores its configuration in JSON files. You can store the configuration either in your user directory or in your workspace, the current project you work.
+VSCode stores its configuration in JSON files. Depending on the scope your
+settings apply to, you can store the configuration either in your user directory
+or in your workspace, the current project you work.
 
-User configuration files are stored in one of the following directories:
+User configuration files apply settings to all project that you open in VScode.
+They are stored in one of the following directories:
 
 * `~/.config/Code/User/` (VSCode)
 * `~/.config/VSCodium/User/` (VSCodium)
 
-Inside your workspace (the project directory), configuration can be stored in a `.vscode` subdirectory.
-
-I prefer to use the per-directory approach which is useful if you
-have checkouts/projects of the same kind in a single directory on the file
-system.
+Workspace configuration files apply settings to the current workspace
+(directory) only and have precedence over user settings. They are stored in the
+`.vscode` subdirectory of the project.
 
 You can find my collection of configuration files for VSCode at
-https://github.com/openSUSE/suse-vscode-doc -- feel free to add you suggestions via GitHub
-Pull Requests or Issues.
+https://github.com/openSUSE/suse-vscode-doc -- feel free to add you suggestions
+via GitHub Pull Requests or Issues.
 
 
 ## XML support
 
-Install the XML extension by RedHat. The exention ID is [`redhat.vscode-xml`](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml).
+Install the XML extension by RedHat. The exention ID is
+[`redhat.vscode-xml`](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml).
 
 ### Support for GeekoDoc's RELAX NG schema
 
 To let VSCode be aware of GeekoDoc, you need to associate XML files with the
-GeekoDoc schema. Add the following to `.vscode/settings.json`:
+GeekoDoc schema. Add the following to `settings.json`:
```

Comment:
It could be helpful (although probably not necessary) to mention where this file should be stored.

```suggestion
GeekoDoc schema. Add the following to `settings.json` and save the file in your
user directory or the project's workspace:
```